docs/UserGuide.md

```diff
@@ -354,13 +355,23 @@ Furthermore, certain edits can cause the Clientell to behave in unexpected ways
 
 --------------------------------------------------------------------------------------------------------------------
 
-## Known issues
+## Known Issues
 
 1. **When using multiple screens**, if you move the application to a secondary screen, and later switch to using only the primary screen, the GUI will open off-screen. The remedy is to delete the `preferences.json` file created by the application before running the application again.
 2. **If you minimize the Help Window** and then run the `help` command (or use the `Help` menu, or the keyboard shortcut `F1`) again, the original Help Window will remain minimized, and no new Help Window will appear. The remedy is to manually restore the minimized Help Window.
 
 --------------------------------------------------------------------------------------------------------------------
 
+## Future Enhancements
+
```

Comment:
I'd prefer the planned enhancement to say fuzzy search instead of partial (because fuzzy is a stronger feature), but this is fine for now -- I'll standardise later